docs/cli/installation.mdx

```diff
@@ -1,10 +1,18 @@
 ---
 title: "Installation"
-description: "Install the Suga CLI"
+description: "Install the Suga CLI and get started with the platform"
 ---
 
 import { OSTabs } from '/snippets/os-tabs.jsx';
 
+## Install Suga CLI
+
+The Suga CLI is all you need to get started with Suga. It provides everything required to build, test, and deploy cloud-native applications with automatic infrastructure generation.
+
+<Note>
```

Comment:
Do we want to mention how/where to request an invite?

Reply:
Done